wrappers/python/openmm/app/internal/amber_file_parser.py

```diff
@@ -461,6 +459,67 @@ def getDihedrals(self):
                                 int(0.5+float(periodicity[iType]))))
         return self._dihedralList
 
+    def getNumMaps(self):
+        """ """
+        try:
+            return self._numCMAP
+        except AttributeError:
+            pass
+        if "CMAP_COUNT" in self._raw_data.keys():
+            self._numCMAP=int(self._raw_data["CMAP_COUNT"][1])
+            return self._numCMAP
+        return False
```

Comment:
Wouldn't it make more sense to return 0 in this case?

Reply:
Returning 0 makes sense.